### DISCUSSION TITLE
Model initialisation

### DISCUSSION THREAD

**Discussion by sahar90o** [Project-MONAI/MONAI#7849] [Q&A]
Hi everyone, 

I am trying to build an ensemble model for 3D segmentation in MONAI. 
to do so I want to train the same model (for example Unet) with different weights. Is there any way to initialize the model weights with different methods using normal, Xavier, Kaimimg, ...? 
Thank you 
Zahra

**Comment by KumoLiu** [comment]
Hi @sahar90o, we have some utilities: https://github.com/Project-MONAI/MONAI/blob/b1c8b42d03bf3c1daabd6c39a35fcbdadb57e8b0/monai/networks/utils.py#L303
And you can also refer to: https://github.com/Project-MONAI/MONAI/blob/08d572839ede5d087e49d4dfadba033e512528df/monai/networks/blocks/localnet_block.py#L286

Hope it helps, thanks.
